svo (install the zip, not me).xml

```diff
@@ -15251,9 +15251,11 @@ svo.valid.remove_unknownany(&quot;clumsiness&quot;)</script>
                         <colorTriggerBgColor>#000000</colorTriggerBgColor>
                         <regexCodeList>
                             <string>You lash out clumsily.</string>
+                            <string>^You lash out at (\w+) with (.+), but miss\.$</string>
```

Comment:
After 5 minutes of bashing with clumsiness on me I did not see a single miss. So I'm guessing it's not making me miss denizens.